functions/4dn-status/src/lambda_function.py

```diff
@@ -33,18 +35,26 @@
 }
 
 
-CALENDAR_DATA_URL = "https://4dn-dcic-publicly-served.s3.amazonaws.com/4dn-status/events.json"
+CALENDAR_DATA_URL_PRD = "https://4dn-dcic-publicly-served.s3.amazonaws.com/4dn-status/events.json"
+CALENDAR_DATA_URL_STG = "https://4dn-dcic-publicly-served.s3.amazonaws.com/4dn-status/events-staged.json"
+
+PRD_ENDPOINT_PATH = '/4dn-status'
+STG_ENDPOINT_PATH = '/4dn-status-staged'
+
+
+def get_calendar_data(staged=False):
+    url = CALENDAR_DATA_URL_STG if staged else CALENDAR_DATA_URL_PRD
 
-def get_calendar_data():
     try:
-        r = requests.get(CALENDAR_DATA_URL)
+        r = requests.get(url)
         result = r.json()
         return result or DEFAULT_DATA
     except Exception:
         return DEFAULT_DATA
 
 
 def convert_to_html(data, environment):
+    ignored(environment)  # TODO: Should this be ignored?
```

Comment:
It's (approximately) an ffenv. In particular, it's the bucket env, so for staging/prod it's been canonicalized to `fourfront-webprod` on the `fourfront` app and to `fourfront-cgap` for `cgap` app.  I don't know that the way in which this conversion occurs needs to care about that, but it could. I left the dataflow just so it's easy to find if it turns out later we care.  Originally I thought it might be used to do filtering of items, but that's now done farther upstream. Even without filtering, though, it _could_ still be used to pull up a different color or icon or something, though. Just probably overkill so I'm not expecting it.

Reply:
And in fact the next comment you made is about the logo thing. So I'll try to connect it to that.

Reply:
Fixed by making it control the icon and header text to be CGAP- or Fourfront-specific.